src/Components/ServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Extensions;
+using System.Collections.Generic;
+
+namespace Components
+{
+	public static class ServiceExtensions
+	{
+		public static IServiceCollection UseComponentSystem(this IServiceCollection services)
```

Comment:
For all the natter, not one person pointed out the obvious Use/Add error. :)  Will fix this too (Use for application extensions in my opinion and Add for service extensions).